app/helpers/application_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def pretty_time(datetime)
```

Comment:
Think of a more suitable name

Reply:
maybe formatted_time would be better here

Reply:
Sure, gonna use formatted_time instead of pretty_time.